legofy/__init__.py

```diff
@@ -48,12 +48,22 @@ def make_lego_image(thumbnail_image, brick_image):
     lego_image = Image.new("RGB", (base_width * brick_width,
                                    base_height * brick_height), "white")
 
+    brick_stats = {}
     for brick_x in range(base_width):
         for brick_y in range(base_height):
             color = rgb_image.getpixel((brick_x, brick_y))
+
+            '''Increment color count'''
```

Comment:
Ok, so im not sure whats going on here completely. But wouldn't this not cover some colors? As in, if some colors in the image aren't in `LEGOS_LABEL`, it wouldn't count them? Also, why are you doing

```
if str(palettes.LEGOS_LABEL[color_label]['rgb']) == str(color):
```

You could just compare the literal r g b values as ints rather than converting to str.